src/main/java/seedu/bigpp/menu/Menu.java

```diff
@@ -0,0 +1,5 @@
+package seedu.bigpp.menu;
+
+public abstract class Menu {
+    
```

Comment:
Consider adding a

`public abstract printMenu(){};`

here, and all the other methods will Override

Reply:
since i need to create non-static methods, i cannot override from menu class